src/seedu/addressbook/data/person/Address.java

```diff
@@ -9,11 +12,16 @@
 public class Address {
 
     public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can only be in a/BLOCK, STREET, UNIT, POSTAL_CODE format";
+    public static final String ADDRESS_VALIDATION_REGEX = "(.+),(.+),(.+),(.+)";
 
-    public final String value;
+    //public final String value;
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.